### PR TITLE
Add space after // comments

### PR DESCRIPTION
**Pull request by kairen**
The commit updated a few comments to keep the code is good syntax.

### CODE REVIEW

**Comment by RA489** [comment]
@minikube-bot OK to test

File: third_party/go9p/ufs_darwin.go [hunk 1]
```diff
@@ -225,7 +225,7 @@ func (u *Ufs) Wstat(req *SrvReq) {
 			case true:
 				mt = st.ModTime()
 			default:
-				//at = time.Time(0)//atime(st.Sys().(*syscall.Stat_t))
+				// at = time.Time(0)//atime(st.Sys().(*syscall.Stat_t))
```
Comment (afbjorklund) [inline third_party/go9p/ufs_darwin.go]: code that is just commented out doesn't normally have spaces (unlike actual comments)

**Comment by kairen** [comment]
> I wonder could this be passed as a goimport flag so this can be done automatically through CI ? and the style be consitent with gofmt instead of personal style?

I trying to find a tool that ensures the comment style is constrained format, but most of the tool(e.g. gofmt, golangci-lint) can't check leading whitespace in comments, because of this format not violate.

**Comment by RA489** [comment]
@minikube-bot OK to test

**Comment by RA489** [comment]
@minikube-bot OK to test

**Comment by RA489** [comment]
@minikube-bot OK to test

**Pull request merged**